app/controllers/idv/capture_doc_controller.rb

```diff
@@ -20,6 +24,13 @@ def return_to_sp
 
     private
 
+    def track_index_loads
+      irs_attempts_api_tracker.idv_phone_upload_link_used(
+        document_capture_session: params[:"document-capture-session"],
```

Comment:
```suggestion
        document_capture_session: params["document-capture-session"],
```

Reply:
can we reuse the method here?
```suggestion
        document_capture_session: document_capture_session_uuid,
```

Reply:
Done - and I reused the `request_id` method, as well.